build.toml

```diff
@@ -29,6 +32,14 @@ target = 'My Computer'
 build_spec = 'Scripting API'
 dependency_target = 'Pharlap'
 
+[[build.steps]]
+name = 'Examples'
+type = 'lvBuildSpec'
+project = '{examples}'
+target = 'My Computer'
+build_spec = 'Examples'
+dependency_target = 'Pharlap'
```

Comment:
```suggestion
```

I don't believe this is needed, although there likely is no harm in having it. We should be building against the API in vi.lib.